website/package.json

```
@@ -29,7 +29,8 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "lint": "eslint --ext .tsx src/",
-    "type-check": "tsc --noEmit"
+    "type-check": "tsc --noEmit",
+    "linting": "eslint src --ext js,ts,tsx,jsx"
```

```suggestion
    "lint": "eslint src --ext js,ts,tsx,jsx"
```
It's more conventional to just call this script lint, so would be nice to write it that way